.gitignore

```diff
@@ -1,2 +1,3 @@
 *.user
 doc/
+WiseTagger.pro.user.6820b68.4.10-pre1
```

Comment:
This should look like `*.user.*` if gitignore can handle it, otherwise each new QtCreator version will generate new files.